invenio_jobs/logging/tasks.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2025 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio jobs logging tasks."""
+
+from celery import shared_task
+from flask import current_app
+from invenio_search import current_search_client
+from invenio_search.utils import prefix_index
+
+
+@shared_task
+def delete_logs():
```

Comment:
you could create a job for it ;) 
that would enable to start the deletion task manually too :)